### DISCUSSION TITLE
DUDA Cuestionario

### DISCUSSION THREAD

**Discussion by Jorge-Macias-Cruz** [LaunchX-InnovaccionVirtual/CursoIntroPython#238] [Q&A] [answered]
Para contestar el cuestionario se tienen que realizar las 9 katas o hasta la 6?

**Comment by Stmark1** [comment]
Lo preferible es que contestes las 10 katas, tienes hasta el lunes 21, si bien lo puedes enviar el enlace ahora e ir contestando las que te faltan eso podria saturarte de actividades, mejor tomalo con calma.

**Comment by FernandaOchoa** [comment, marked as answer]
Hola, son 10 katas las que debes contestar, sin embargo si consideras que puedes finalizarlas esta semana, te recomiendo llenar el formulario y posteriormente ir actualizando tu repositorio. Conforme sientas más seguridad para irlas respondiendo. Entiendo que no todes aprenden de la misma manera, por lo que hay quienes prefieren garantizar su lugar respondiendo el formulario con lo que tienen y continuar con sus aprendizajes con toda calma y a su ritmo.

Mi recomendación es no presionarse, por que la ventaja de GitHub es que puedes proporcionar el link y seguir trabajando, con tu lugar asegurado en la siguiente misión.
